homeassistant/components/hvv_departures/config_flow.py

```diff
@@ -0,0 +1,224 @@
+                    step_id="station", data_schema=SCHEMA_STEP_STATION, errors=errors
+                )
+            except CannotConnect:
+                errors["base"] = "cannot_connect"
+            except InvalidAuth:
+                errors["base"] = "invalid_auth"
+
+        return self.async_show_form(
+            step_id="user", data_schema=SCHEMA_STEP_USER, errors=errors
+        )
+
+    async def async_step_station(self, user_input=None):
+        """Handle the step where the user inputs his/her station."""
+        errors = {}
+        if user_input is not None:
+
+            check_name = await self.hub.gti.checkName(
+                {"theName": {"name": user_input["station"]}, "maxList": 20}
+            )
+
+            results = check_name.get("results")
+
+            if not results:
+                errors["base"] = "no_results"
+
+                return self.async_show_form(
+                    step_id="station", data_schema=SCHEMA_STEP_STATION, errors=errors
+                )
+
+            self.stations = {
+                "{} ({})".format(x.get("name"), x.get("type")): x for x in results
+            }
+
+            schema = vol.Schema(
+                {vol.Required("station"): vol.In(list(self.stations.keys()))}
+            )
+
+            return self.async_show_form(
+                step_id="station_select", data_schema=schema, errors=errors
+            )
+
+        return self.async_show_form(
+            step_id="station", data_schema=SCHEMA_STEP_STATION, errors=errors
+        )
+
+    async def async_step_station_select(self, user_input=None):
+        """Handle the step where the user inputs his/her station."""
+        errors = {}
+        if user_input is not None:
+            self.data.update({"station": self.stations[user_input["station"]]})
+
+            # get station information
+            station_information = await self.hub.gti.stationInformation(
+                {"station": self.data["station"]}
+            )
+
+            self.data.update({"stationInformation": station_information})
+
+            title = self.data["station"]["name"]
+
+            return self.async_create_entry(title=title, data=self.data)
+
+        return self.async_show_form(
+            step_id="station_select", data_schema=SCHEMA_STEP_STATION, errors=errors
+        )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Get options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Options flow handler."""
+
+    def __init__(self, config_entry):
+        """Initialize HVV Departures options flow."""
+        self.config_entry = config_entry
+        self.options = dict(config_entry.options)
+        self.filters = {}
+        self.hub = None
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        errors = {}
+        if self.filters == {}:
```

Comment:
Use it as a falsy value:
```suggestion
        if not self.filters:
```